main.cpp

```diff
@@ -0,0 +1,166 @@
+#include "City.h"
+using namespace std;
+const int N = 4;
```

Comment:
Please make clearer what is N and where you use it.

Reply:
use in array for sorting , changing if changed number of films.(added coment)